### PR TITLE
Closes #307 | Implement dataloader for ViSoBERT 

### LINKED ISSUE #307: Create dataset loader for ViSoBERT
Dataloader name: `visobert/visobert.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?visobert

|     Dataset| visobert |
|-------------|---|
| Description | The ViSoBERT is textual data crawled from three most well-known Vietnamese public social networks (Facebook, TikTok, and YouTube) by research API of these platform. The dataset contains Facebook posts, TikTok comments, and Youtube comments of Vietnamese-verified users, from Jan 2016 (Jan 2020 for TikTok) to Dec 2022. A post-processing mechanism is applied to handles hashtags, emojis, misspellings, hyperlinks, and other noncanonical texts. |
| Subsets | - |
| Languages | vie |
| Tasks | Language Modeling |
| License | Creative Commons Attribution Non Commercial 4.0 (cc-by-nc-4.0) |
| Homepage | https://drive.google.com/drive/folders/1C144LOlkbH78m0-JoMckpRXubV7XT7Kb |
| HF URL | https://huggingface.co/uitnlp/visobert |
| Paper URL | https://aclanthology.org/2023.emnlp-main.315.pdf |

### PR DESCRIPTION
**Pull request by raileymontalan**
Closes #307

### Checkbox
- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `seacrowd/sea_datasets/visobert/visobert.py` (please use only lowercase and underscore for dataset naming).
- [x] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_SEACROWD_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `SEACrowdConfig` for the source schema and one for a seacrowd schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_seacrowd seacrowd/sea_datasets/<visobert>/<visobert>.py`.
- [x] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

### CODE REVIEW

**Review by holylovenia — APPROVED** [review]
LGTM!! Thank you @raileymontalan for your contribution. 😁

**Review by akhdanfadh — COMMENTED** [review]
Tested and worked, only one main discussion from me.

File: seacrowd/sea_datasets/visobert/visobert.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    def _info(self) -> datasets.DatasetInfo:
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "text": datasets.Value("string"),
+                }
+            )
+        elif self.config.schema == "seacrowd_ssp":
+            features = schemas.self_supervised_pretraining.features
```
[the remark this answers is not in the document]
Comment (raileymontalan) [reply seacrowd/sea_datasets/visobert/visobert.py:119]: Will combine these in the next commit. Thanks!

File: seacrowd/sea_datasets/visobert/visobert.py [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+            elif self.config.schema == "seacrowd_ssp":
+                for idx, row in enumerate(f):
+                    if row.strip() != "":
+                        yield (
+                            idx,
+                            {
+                                "id": str(idx),
+                                "text": row.strip(),
+                            },
+                        )
```
[the remark this answers is not in the document]
Comment (raileymontalan) [reply seacrowd/sea_datasets/visobert/visobert.py:173]: Will combine these in the next commit. Thanks!

**Review by akhdanfadh — COMMENTED** [review]
LGTM, tested and worked. Thanks for contributing @raileymontalan! Merging this.

**Pull request merged**